Gds/src/fprime_gds/common/adapters/ip.py

```diff
@@ -77,14 +76,13 @@ def open(self):
             self.thudp.start()
             # Start up a keep-alive ping if desired. This will hit the TCP uplink, and die if the connection is down
             if IpAdapter.KEEPALIVE_INTERVAL is not None:
-                self.keepalive = threading.Thread(
```

Comment:
Why did you remove this keep-alive thread?

Reply:
I deleted the variable because threading.Thread(...).start() doesn't have a return value and the vairavle was. Thus pylint was complaining that it was "Assigning result of a function call, where the function has no return". The key here is the .start() has no return, but 'threading.Thread()' does. Therefore the correct solution is as follows
```
self.keepalive = threading.Thread(
    target=self.th_alive, args=[float(self.KEEPALIVE_INTERVAL)]
)
self.keepalive.start()
```